packages/jest/test/__snapshots__/preact.test.js.snap

```diff
@@ -1,21 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`jest-emotion with preact handles elements with no props 1`] = `"<div />"`;
-
-exports[`jest-emotion with preact replaces class names and inserts styles into preact test component snapshots 1`] = `
-".emotion-0 {
-  color: red;
-}
-
-.emotion-1 {
-  width: 100%;
-}
```

Comment:
It looks like those were actually working :p although I guess that there is no special handling for this in our code?

Reply:
Lol, so yes the snapshots were working, though we were on a very old version of Preact and using `preact-render-to-json` which is now defunct AFAIK.

I guess, bottom line is, I don't see the need for Preact tests if Emotion doesn't support Preact.